parseabc.c

```diff
@@ -2748,7 +2750,7 @@ parsemusic (field)
                  }
               else
 	        event_split_voice ();
-	        break;
+	      break;
```

Comment:
This preserves the original semantics and fixes the indentation.  If /* fall through */ was intended, there should be a comment to that effect.